MessageBird/Objects/Message.cs

```diff
@@ -27,7 +27,9 @@ public enum MessageType
         [EnumMember(Value = "premium")]
         Premium,
         [EnumMember(Value = "flash")]
-        Flash
+        Flash,
+        [EnumMember(Value ="tts")]
+        tts
```

Comment:
This enum is also used by the `Message` object, which does not support `"type": "tts"`. 

Two easy solution are:
1) Creating a new enum, specific to `verify`;
2) Throwing an exception when trying to create a message with type `MessageType.Tts`.

I would strongly prefer the second option. The first may be neater, but would be backwards incompatible and hence require a major version bump, which is not on the roadmap at this moment.

What do you think?